### PR TITLE
README: document development dependencies for Fedora/Ubuntu

### PR DESCRIPTION
**Pull request by jelly**
Document what is required to get a test VM build done, for some new contributors the gettext dependency is not obvious to resolve.

See for example https://github.com/cockpit-project/cockpit-podman/pull/1218

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,19 @@
...
 
+# Development dependencies
+
+On Debian/Ubuntu:
+
+    $ sudo apt install gettext nodejs npm make
+
+On Fedora:
+
+    $ sudo dnf install gettext nodejs npm make
```
Comment (martinpitt) [inline README.md:13]: No `npm` please -- we have tools/node-modules which runs this in a container.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (martinpitt) [reply README.md:13]: Nonsense -- I was thinking `npm install`, but of course we also need it for `npm run`. Not strictly (we could call it directly as well), but our Makefile does `npm run watch`.

**Review by martinpitt — APPROVED** [review]
Thanks! Please trigger required tests. In general, no-test does not  really make sense for starter-kit.

**Comment by jelly** [comment]
> Thanks! Please trigger required tests. In general, no-test does not really make sense for starter-kit.

I thought it would be good to limit CI usage

**Pull request merged**